aomi/template.py

```diff
@@ -68,3 +72,51 @@ def load_var_files(opt):
         obj = merge_dicts(obj.copy(), yamlz)
 
     return obj
+
+
+def load_template_help(builtin):
+    """Loads the help for a given template"""
+
+    help_file = "templates/%s-help.yml" % builtin
+    help_file = resource_filename(__name__, help_file)
+    help_obj = {}
+    if os.path.exists(help_file):
+        help_data = yaml.load(open(help_file))
+        if 'name' in help_data:
+            help_obj['name'] = help_data['name']
+
+        if 'help' in help_data:
+            help_obj['help'] = help_data['help']
+
+        if 'args' in help_data:
+            help_obj['args'] = help_data['args']
+
+    return help_obj
+
+
+def builtin_list():
+    """Show a listing of all our builtin templates"""
+    for template in resource_listdir(__name__, "templates/"):
```

Comment:
That's right... Should be using https://github.com/Autodesk/aomi/blob/master/aomi/helpers.py#L45